Objects/longobject.c

```diff
@@ -695,6 +695,13 @@ _PyLong_Sign(PyObject *vv)
     return Py_SIZE(v) == 0 ? 0 : (Py_SIZE(v) < 0 ? -1 : 1);
 }
 
+static int
+_Py_bit_length_digit(digit x)
```

Comment:
Since the scope of this function is just file, _Py_ prefix must be removed:

```suggestion
bit_length_digit(digit x)
```